src/Wallabag/CoreBundle/Helper/EntriesExport.php

```diff
@@ -73,6 +73,31 @@ public function updateTitle($method)
     }
 
     /**
+     * Sets the author for one entry or category.
+     *
+     * The publishers are used, or the domain name if empty.
+     *
+     * @param string $method Method to get articles
+     *
+     * @return EntriesExport
+     */
+    public function updateAuthor($method)
+    {
+        $this->author = $method.' authors';
+
+        if ('entry' === $method) {
+            $publishedBy = $this->entries[0]->getPublishedBy();
+            if (!empty($publishedBy)) {
+                $this->author = implode(', ', $this->entries[0]->getPublishedBy());
+            } else {
+                $this->author = $this->entries[0]->getDomainName();
```

Comment:
You can avoid the `else` by redefining `$this->author = $this->entries[0]->getDomainName();` before the `if`.

Also, I think we can return early to avoid cascading spaces by doing:

```php
if ('entry' !== $method) {
    $this->author = $method.' authors';
    return $this;
}

// then do the job about the entry method
```